### PR TITLE
fix: some minor edits

### PR DESCRIPTION
**Pull request by sublimator**

### CODE REVIEW

File: static/specification.html [hunk 1]
```diff
@@ -302,7 +302,7 @@ <h2>
           <dfn>Payment end-point</dfn>:
         </dt>
         <dd>
-          A [=URL=] to a <a data-cite="SPSP#specification">SPSP end-point</a>
+          An [=URL=] to a <a data-cite="SPSP#specification">SPSP end-point</a>
```
Comment (marcoscaceres) [inline static/specification.html:305]: See "a URL" is correct as per https://url.spec.whatwg.org
```suggestion
          A [=URL=] to a <a data-cite="SPSP#specification">SPSP end-point</a>
```

**Review by marcoscaceres — APPROVED** [review]
Couple of nits/suggestion! thanks @sublimator!!!

File: static/specification.html [hunk 2]
```diff
@@ -693,7 +693,7 @@ <h2>
           <dfn>paymentPointer</dfn> attribute
         </h2>
         <p>
-          A URL representing a [=payment end-point=]. When getting, returns the
+          An URL representing a [=payment end-point=]. When getting, returns the
```
[the remark this answers is not in the document]
Comment (sublimator) [reply static/specification.html:696]: I guess you read it as `y`oo-are-ell with a consonant leading sound

File: static/specification.html (hunk 1 above)
Comment (sublimator) [reply static/specification.html:305]: Yeah, I guess so, I'd stomach it better if it was spelled U.R.L :)

**Pull request merged**